universal-login-wallet/cypress/support/loginAppCommands.ts

```diff
@@ -10,7 +10,9 @@ Cypress.Commands.add('pickUsername', (name) => {
   cy.get('.unilogin-component-suggestions-list').first()
     .get('.unilogin-component-suggestions-ens-name')
     .get('p.unilogin-component-ens-name').contains(name);
-  cy.get('.unilogin-component-suggestions-list').first().find('#create-new').click();
+  cy.get('.unilogin-component-suggestions-list').first().find('#create-new').click().should(() => {
+    expect(localStorage.getItem('wallet-ganache.kind')).to.eq = 'Future4';
```

Comment:
Do we need this `4`? 

Also, I don't think that we should put here `expect`. This file contains the setup for tests and shouldn't be a test itself.